tests/test_core/test_patch_chunk.py

```
@@ -304,9 +307,20 @@ def patches_conflicting_private_coord(self, random_patch):
+        """Unequal distance coords partition rather than raise (0.2 change).
+
+        Patches whose non-chunked dimension coordinates differ are never
+        combined; they simply land in separate output patches.
+        """
+        out = distance_adjacent.chunk(time=...)
+        assert len(out) == len(distance_adjacent)
+
+        # the differing distance envelopes are preserved, not merged/duplicated
+        def _distance_envelopes(spool):
+            df = spool.get_contents()
+            return sorted(zip(df["distance_min"], df["distance_max"]))
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)